src/vdom/diff.js

```diff
@@ -187,10 +187,10 @@ function innerDiffNode(dom, vchildren, context, mountAll, absorb) {
 		min = 0,
 		len = originalChildren.length,
 		childrenLen = 0,
-		vlen = vchildren && vchildren.length,
+		vlen = vchildren ? vchildren.length : 0,
```

Comment:
Thanks for your insights, @bmeurer!

Would an `undefined` default (e.g.`vlen = vchildren ? vchildren.length : undefined`) be less optimal for the compiler than the default of `0`? Same question for `null`. Thanks!